tox.ini

```diff
@@ -177,6 +177,9 @@ deps =
   elasticsearch6: elasticsearch>=6.0,<7.0
   elasticsearch7: elasticsearch-dsl>=7.0,<8.0
   elasticsearch7: elasticsearch>=7.0,<8.0
+  sqlalchemy13: sqlalchemy~=1.3,<1.4
```

Comment:
can we change this to 1.2 or 1.1 so we test oldest reasonably supported version and the newest (1.4)?

Reply:
Updated to use 1.1.